enterprise/internal/insights/background/historical_enqueuer.go

```
@@ -364,7 +370,17 @@ func (h *historicalEnqueuer) Handler(ctx context.Context) error {
 	// are filtered at view time of an insight.
 	ctx = actor.WithInternalActor(ctx)
 
-	// Discover all insights on the instance.
+	convertJITInsights := true
+	deprecateJITInsights, _ := h.featureFlagStore.GetFeatureFlag(ctx, "code_insights_deprecate_jit")
```

This method returns an error if the feature flag does not exist.  Ignored error since default mode is to not have the flag and logging the error is not useful.